### PR TITLE
Jaegle2021perceiver 1

### PR DESCRIPTION
**Pull request by aabayomi**

### CODE REVIEW

File: _summaries/jaegle2021perceiverio_1.md [hunk 1]
```diff
@@ -1,19 +1,66 @@
...
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+### What is the core idea?
+
+The authors proposes a perceiver model that scales linearly in computation on varying inputs sizes by encoding its inputs into smaller latent arrays achieving SOTA results .
```
Comment (timchen0618) [inline _summaries/jaegle2021perceiverio_1.md:13]: perceiverIO model
scale linearly on both input/output size

File: _summaries/jaegle2021perceiverio_1.md [hunk 2]
```diff
@@ -1,19 +1,66 @@
...
+<img width="700px" src="jaegle2021perceiverio_1f.png"/>
+
+
+- Inputs are __encoded__ to a latent space using the attention module
+
+- The latent representation is refined via many layers of __processing__
+
+- The latent space is __decoded__ to produce outputs using attention module
+
+The attention modules combines both a global query-key-value (QKV) operation and a multi-layer perceptron (MLP) operation.Similar to transformer but efficient in compute and memory.The model produces an output query matrix, O, that varies with the input size depending on the task.computational complexity of each attention module in QKV attention,is O(M N F ) for two matrix multiplication of  dimension M x F and M x N,with L latent attention blocks has complexity O([M + O + LN ]N F ).
```
Comment (timchen0618) [inline _summaries/jaegle2021perceiverio_1.md:27]: I think O varies with output size?

File: _summaries/jaegle2021perceiverio_1.md [hunk 3]
```diff
@@ -1,19 +1,66 @@
...
+
+### How well does the paper perform?
+
+<img width="700px" src="jaegle2021perceiverio_1e.png"/>
+
+ Perceiver performed better than compared transformer models on GLUE benchmarks.
```
Comment (timchen0618) [inline _summaries/jaegle2021perceiverio_1.md:33]: Perceiver -> PerceiverIO
could also mention that UTF-8 byte model achieved great results without tokenization (which is important preprocessing step in NLP)

File: _summaries/jaegle2021perceiverio_1.md (hunk 2 above)
Comment (timchen0618) [reply _summaries/jaegle2021perceiverio_1.md:27]: Could talk about how the queries are created since they are influential to model performance

File: _summaries/jaegle2021perceiverio_1.md [hunk 4]
```diff
@@ -1,19 +1,66 @@
...
+
+### What interesting variants are explored?
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+
+* Perceivers has a faster computation time compared to a transformer
+
```
Comment (timchen0618) [inline _summaries/jaegle2021perceiverio_1.md:64]: could add that it achieve great results on various domain and tasks, and has the potential to be a general architecture that can solve nearly any task in Deep Learning

File: _summaries/jaegle2021perceiverio_1.md [hunk 5]
```diff
@@ -1,19 +1,66 @@
...
+- Produced an output feature grid with stride 4 and 64 channels
+
+### What interesting variants are explored?
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+
+* Perceivers has a faster computation time compared to a transformer
```
Comment (timchen0618) [inline _summaries/jaegle2021perceiverio_1.md:63]: could add that its linear to input/output size
